lib/travis/build/addons/postgresql.rb

```diff
@@ -9,13 +9,17 @@ class Postgresql < Base
 
         def after_prepare
           sh.fold 'postgresql' do
-            sh.export "PATH", "/usr/lib/postgresql/#{version}/bin:$PATH", echo: false
+            sh.export 'PATH', "/usr/lib/postgresql/#{version}/bin:$PATH", echo: false
             sh.echo "Starting PostgreSQL v#{version}", ansi: :yellow
-            sh.cmd "service postgresql stop", assert: false, sudo: true, echo: true, timing: true
+            sh.cmd 'service postgresql stop', assert: false, sudo: true, echo: true, timing: true
             sh.if "-d /var/ramfs && ! -d /var/ramfs/postgresql/#{version}", echo: false do
               sh.cmd "cp -rp /var/lib/postgresql/#{version} /var/ramfs/postgresql/#{version}", sudo: true, assert: false, echo: false, timing: false
             end
             sh.cmd "service postgresql start #{version}", assert: false, sudo: true, echo: true, timing: true
+            %w(5432 5433).each do |pgport|
```

Comment:
Take a peek at the PR description :wink:

I gave the ports names for clarity, too.